lib/maplibre_gl.dart

```diff
@@ -43,6 +43,7 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
+import 'package:maplibre_gl/maplibre_gl.dart';
```

Comment:
This seems wrong, isn't this the parent?

Reply:
Woops, good point; removed it.